.github/workflows/push_monitoring_image.yaml

```diff
@@ -0,0 +1,43 @@
+name: build_push_metrics_scraper_image
+
+on:
+  push:
+    branches: 
+    - main
+  # Setting this enables manually triggering workflow in the GitHub UI
+  # see https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch: {}
+
+permissions: read-all
+
+# Build and push the monitoring image.
+jobs:
+  build_civiform_cloud_deployment:
+    runs-on: ubuntu-latest
+
+    concurrency:
+      group: build-${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
+    name: Build civiform-cloud-deployment
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Set up Docker QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
**question (non-blocking):** is this necessary? we don't use it in our other workflows for building images

Reply:
Removed!